paper/paper.md

```diff
@@ -0,0 +1,51 @@
+---
+title: 'LangToGroup: A Haskell package for group presentation construction from formal language'
+tags:
+  - Haskell
+  - formal languages
+  - group theory
+  - Turing machines
+  - group presentation construction
+authors:
+  - name: Semyon Grigorev
```

Comment:
I'm not sure that I should be the first. Project lead traditionally last in the list of authors.